e2e/infra/app_func.go

```diff
@@ -0,0 +1,16 @@
+package infra
+
+import (
+	"net/http"
+	"net/url"
+)
+
+// Login logins to the bookinfo app and returns the cookies
+func Login(rawURL, user, password string) (string, []*http.Cookie, error) {
```

Comment:
How about naming it `bookinfo_interactions.go` or something between these lines? The intention would be to gather all bookinfo testing code in one place. WDYT?